index.js

```diff
@@ -86,69 +88,80 @@ module.exports = (app) => {
           `\n\n${summary}`
       }
 
-      await context.github.checks
-        .create(
-          context.repo({
-            name: 'DCO',
-            head_branch: pr.head.ref,
-            head_sha: pr.head.sha,
-            status: 'completed',
-            started_at: timeStart,
-            conclusion: 'action_required',
-            completed_at: new Date(),
-            output: {
-              title: 'DCO',
-              summary
-            },
-            actions: [
-              {
-                label: 'Set DCO to pass',
-                description: 'would set status to passing',
-                identifier: 'override'
+      if (enablePass) {
```

Comment:
this if statement makes sense, but there's no `else` later on. If you don't have an else and include an api call without the `actions` block, no failing check will get created at all

Reply:
actions block referring to https://github.com/probot/dco/pull/120/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R106-R112

Reply:
Good point! Added that now - let me know if the status returned makes sense.